src/storage/src/hummock/state_store.rs

```diff
@@ -272,6 +274,7 @@ impl StateStore for HummockStorage {
     fn clear_shared_buffer(&self) -> impl Future<Output = StorageResult<()>> + Send + '_ {
         self.min_current_epoch
```

Comment:
Will this comment be fixed by the way in this PR? Or we have another PR?

Here I think we should at least set the `seal_epoch` as the epoch in hummock version, and otherwise the CN will not be able to serve uncommitted read.

Reply:
I'm sorry but I have no clear idea how to fix this. 🥵 Would appreciate help from you storage experts.

Reply:
In this PR we can do the following instead
```
self.seal_epoch.store(
    self.pinned_version.load().max_committed_epoch(),
    MemOrdering::SeqCst,
)
```

Reply:
Done, PTAL. 🥰